.vsts-ci.yml

```diff
@@ -0,0 +1,56 @@
+phases:
+- template: /eng/build.yml
+  parameters:
+    agentOs: Windows_NT
+    queue:
+      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+        name: Hosted VS2017
+      ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:        
+        name: DotNetCore-Build
+        demands:
+        - agent.os -equals Windows_NT        
+      parallel: 2
+      matrix:
+        Build_Debug:
+          _BuildConfig: Debug
+          _PublishType: none
+          _SignType: test
+        Build_Release:
+          _BuildConfig: Release
+          ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+            _PublishType: none
+            _SignType: test
+            PB_PublishBlobFeedKey:
+            PB_PublishBlobFeedUrl:
+          ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+            _PublishType: blob
+            _SignType: real
+
+
+- template: /eng/build.yml
+  parameters:
+    agentOs: Linux
+    dockerImage: microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
+    queue: 
+      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+        name: DotNetCore-Linux
+      ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:        
+        name: DotNetCore-Test
+      demands:
+        - agent.os -equals Linux
+      parallel: 2
+      matrix:
+        Build_Debug:
+          _BuildConfig: Debug
+          _PublishType: none
+          _SignType: test
+        Build_Release:
+          _BuildConfig: Release
+          _PublishType: none
+          ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+            _SignType: test
+            PB_PublishBlobFeedKey:
```

Comment:
You could make `PB_PublishBobFeedKey` and `PB_PublishBlobFeedUrl` global variables.  ie. move these to the top of the file.

```YAML
variables:
  ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
    PB_PublishBlobFeedKey:
    PB_PublishBlobFeedUrl: 

phases:
- template: /eng/build.yml
…
```

Reply:
Cool. Will do.